examples/compiled/bar_corner_radius_end.vg.json

```diff
@@ -49,6 +49,7 @@
   "marks": [
     {
       "type": "group",
+      "name": "stackgroup",
```

Comment:
Why do we need to add all of these "stackgroup"?

Reply:
I don't remember the reason I put it there (I might saw it from pathgroup). I will remove it.